src/api/routes/experiment.js

```diff
@@ -12,6 +13,14 @@ module.exports = {
         .catch(next);
     },
   ],
+  'experiment#createExperiment': [
+    expressAuthorizationOnlyMiddleware,
```

Comment:
This should be called something like `expressAuthenticationOnlyMiddleware` -- makes it more generic and the `Only` ensures that people notice the difference between `authorization` and `authentication`.